File: generare-numere-fibonacci/fisier.java
import java.util.Scanner;
class Tutorial {
    // afiseaza primele n numere fibonacci
    public static void generare_fibonacci(int n) {
        int a = 1, b = 1;
        System.out.println(a);
        System.out.println(b);
        for (int i = 1; i <= n; ++i) {
            int c = a + b;
            System.out.println(c);
            a = b;
            b = c;
        }
    }
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        generare_fibonacci(n);
    }
}

File: generare-numere-fibonacci/fisier.cpp
#include <iostream>
using namespace std;
// afiseaza primele n numere fibonacci
void generare_fibonacci(int n) {
    int a = 1, b = 1;
    cout << a << "\n" << b << "\n";
    for (int i = 1; i <= n; ++i) {
        int c = a + b;
        cout << c << "\n";
        a = b;
        b = c;
    }
}
int main() {
    int n; cin >> n;
    generare_fibonacci(n);
    return 0;
}
